ReleaseNotes.md

```diff
@@ -1,5 +1,36 @@
 # Release Notes
 
+## New in Release 2.8.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - `nncf.quantize` signature has been changed to add `mode: Optional[nncf.QuantizationMode] = None` as its 3-rd argument, between the original `calibration_dataset` and `preset` arguments.
```

Comment:
Why is the first bullet relates to breaking changes? It looks like an extension of the current API but not the change. I mean the existing users' calibration pipeline will not be broken with this change after update of the NNCF version.

Reply:
I agree and I would mention it in the bullet about FP8 below

Reply:
The existing pipelines will be broken for the following use case:
```
quantized_model = nncf.quantize(model, dataset, QuantizationPreset.Performance)
```
and basically any usage where more than 2 of the `nncf.quantize` arguments have been used positionally.

Reply:
I agree to keep it here if it really breaks the BC. But maybe it is worth adding info on how the code should be changed, e.g.:
Use `quantized_model = nncf.quantize(model, dataset, mode=QuantizationMode.PERFORMANCE)` instead of Use `quantized_model = nncf.quantize(model, dataset, QuantizationPreset.PERFORMANCE)`.

Reply:
has anyone tried to reproduce it to ensure we broke the BC here?
cc @alexsu52 @KodiaqQ

Reply:
I've tested locally on the `nncf/examples/post_training_quantization/openvino/mobilenet_v2/main.py`. 
The scenario with the `nncf.quantize(model, dataset, QuantizationPreset.PERFORMANCE)` would throw:
`RuntimeError: overflow_fix option with OverflowFix.FIRST_LAYER value is not supported with the mode option!`. 
Because the mode is not `None` other options must be disabled as well.